best-sum/best_sum.py

```diff
@@ -1,4 +1,32 @@
 def best_sum(target_sum, numbers, memo = {}):
   ''' Returns the shortest array of elements that add upto the target sum '''
+  
+  def best_sum(target, numbers, memo=None):
```

Comment:
It appears that this def is nested inside best_sum? please ensure that you are not overloading the method name space and that your tabbing is accurate.